bikespace_frontend/src/components/landing/content-blocks/content-blocks.module.scss

```diff
@@ -56,15 +59,28 @@ a.buttonLink {
   &:hover {
     border-color: $color-primary;
   }
+  padding: 1.5rem;
```

Comment:
This rule on line 62 doesn't need to be added again since it's already specified above; for lines 63-64, I would move above the & specifiers